final_task/rss_reader/printers.py

```diff
@@ -0,0 +1,118 @@
+import json
+import pprint
+import logging as log
+
+from collections import OrderedDict
+from collections import defaultdict
+from colorama import Fore
+from colorama import Style
+
+import rss_get_items as filters
+
+
+def split_string_by_lines(input_string: str, word_number: int) -> str:
+    input_string = input_string.strip().split()
+    result = ''
+    for idx in range(len(input_string) // word_number + 1):
+        start = idx * word_number
+        end = (idx + 1) * word_number
+        result += ' '.join(input_string[start: end]) + '\n'
+    return result.strip()
+
+
+def prepare_one_item(item_xml: defaultdict) -> OrderedDict:
+    """"
+
+    Take one rss item as dictionary and make ordered dict
+    with title, date, description and media content
+
+    """
+    title = filter_title(item_xml['title'])
+    date = (item_xml['pubDate'])
+    date = data_split(date)
+    news_link = item_xml['link']
+    description = split_string_by_lines(
+        filters.description(item_xml['description']),
+        11
+    )
+    media_content = split_string_by_lines(
+        ' '.join(item_xml['content']),
+        1
+    )
+    prepared_news = OrderedDict()
+
+    prepared_news["Title:"] = title
+    prepared_news["Date:"] = date
+    prepared_news["Link: "] = news_link
+    prepared_news["Description: "] = description
+    prepared_news["Media content:\n"] = media_content
+    return prepared_news
+
+
+def filter_title(title: str) -> str:
+    new_title = title.replace("&#39;", '')
```

Comment:
It is better to have a bit more universal solution, as there might be other troublesome symbols.